added docs for forestty migration updates (#1470)
tinacms/tina.io: content/docs/forestry/common-errors.md

## Common error message and how to fix them

### GraphQL error

```
Error: GraphQL Document Validation failed with 11 errors;
  Error 0: Fields "<FieldName>" conflict because they return conflicting types "String!" and "String". Use different aliases on the fields to fetch both if this was intentional.
```

This error is because two fiends in your schema have the same name with different types and when querying for both GraphQL throws an error. This error occurs when generating the GraphQL client so if you are not using our data fetching you can mitigate this error by passing the `--noSDK` option to both `tinacms dev` and `tinacms build`.

You can [read more about this issue here](https://github.com/tinacms/tinacms/issues/3150#issuecomment-1239796350)

### Error when getting content in the admin

```
[Error] GetCollection failed: Unable to fetch, errors: Error querying file <File> from collection pages. Please run "tinacms audit" or add the --verbose option for more info
```

![](https://res.cloudinary.com/forestry-demo/image/upload/v1673619483/tina-io/docs/forestry-migration/Screen-Shot-Error-Messager.png)

> This can happen for many reason but a common reason is because the data you are trying to fetch is not in the correct format, missing a required field or is missing the `_template` key. If you look at the error message in the terminal you should see the exact error that is causing the issue.

Reviewer: ```suggestion
This can happen for many reasons. Some common reasons are because the data you are trying to fetch is 
- not in the correct format 
- missing a required field
- is missing the `_template` key. 
If you look at the error message in the terminal you should see the exact error that is causing the issue.
```

For example, if you are missing `_template` you will see an error like this

```
Error: Unable to determine template for item at hugo/content/about.md, no template name provided for collection with multiple templates
    at TinaSchema.getCollectionAndTemplateByFullPath (/Users/logananderson-forestry-mac/dev/forestry.io/forestry.io/node_modules/@tinacms/schema-tools/dist/index.js:224:19)
```

This can be fixed by adding the `_template` key to the front matter of the file.

```md
---
# ...
_template: my_template
---
```

> Where "my_template" is the name of the template in the collection to which the file belongs.

Reviewer: ```suggestion
..where "my_template" is the name of the template in the collection to which the file belongs.
```
